packages/eslint-plugin/src/configs/react.js

```diff
@@ -27,6 +37,14 @@ const config = {
         project: ['./tsconfig.json', path.join(gitRoot, 'tsconfig.json')],
       },
     },
+    jsdoc: {
+      ignoreInternal: true,
+      tagNamePreference: {
+        defaultvalue: 'defaultvalue',
+        defaultValue: 'defaultValue',
+        return: 'return',
+      },
```

Comment:
These are temporary and will be removed in a future PR. I wanted to scope this PR to adding the new plugin, and a future PR to fix places that use `@defaultvalue`

Reply:
can we move to this to generic config ? every platform should use same conventions (node/web)

Reply:
Right now the node config actually extends from this one and just turns off `no-console`